client/src/Pages/Home/index.js

```diff
@@ -0,0 +1,175 @@
+import ImageCard from '../../Components/Common/ImageCard';
+import Header from '../../Components/Common/Header';
+import SearchBox from '../../Components/Common/SearchBox';
+import NavBar from '../../Components/Common/Navbar';
+import './style.css';
+
+const axios = require('axios');
+
+function Home() {
+  const [brands, setBrands] = useState([]);
+  const [types, setTypes] = useState([]);
+  const [review, setReview] = useState([]);
+  const getCarsBrands = async () => {
+    try {
+      const data = await axios.get('/api/v1/brands');
+      setBrands(data.data.data);
+    } catch (err) {
+      console.log(err);
+    }
+  };
+  const getTCarsType = async () => {
+    try {
+      const data = await axios.get('/api/v1/types');
+      setTypes(data.data);
+    } catch (err) {
+      console.log(err);
+    }
+  };
+  useEffect(() => {
+    getTCarsType();
+  }, []);
+  useEffect(async () => {
+    getCarsBrands();
+  }, []);
+  useEffect(async () => {
+    try {
+      const data = await axios.get('/api/v1/reviewHome');
+      setReview(data.data.data);
+    } catch (err) {
+      console.log(err);
+    }
+  }, []);
+  const Item = styled(Paper)(({ theme }) => ({
+    ...theme.div,
+    width: '100%',
+  }));
+  return (
+    <div>
+      <Header img="https://images-ext-2.discordapp.net/external/fKDzIa5WJukHd7AapdproCqAthYTKUcaIHmE_M2_VMM/%3Fixlib%3Drb-1.2.1%26ixid%3DMnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8%26auto%3Dformat%26fit%3Dcrop%26w%3D870%26q%3D80/https/images.unsplash.com/photo-1449965408869-eaa3f722e40d?width=641&height=427">
+        <div className="header">
+          <NavBar />
+          <Typography variant="h2" gutterBottom className="paragraph-description" sx={{ fontSize: '40px' }}>
+            First Class Car Rental & Limousine Services
+          </Typography>
+          <Typography variant="body1" gutterBottom className="paragraph-description" sx={{ fontSize: '24px', paddingLeft: '25px' }}>
+            We offer professional car rental & limousine services in our range of high-end vehicles.
+          </Typography>
+          <SearchBox />
+        </div>
+      </Header>
+      <Typography variant="h5" gutterBottom className="paragraph-description" sx={{ textAlign: 'center', paddingTop: '50px' }}>
+        Find Car by Brands
+      </Typography>
+      <Typography
+        variant="body1"
+        gutterBottom
+        className="paragraph-description"
+        sx={{
+          textAlign: 'center', paddingTop: '2px', color: '8A8A8A', paddingBottom: '25px',
+        }}
+      >
+        We offer professional car rental & limousine services in our range of high-end vehicles.
+      </Typography>
+      <Box sx={{
+        flexGrow: 1, marginLeft: '3%', marginRight: '3%', height: '600px', paddingTop: '15px',
+      }}
+      >
+        <Grid
+          container
+          spacing={{ xs: 2, md: 3 }}
+          columns={{ xs: 4, sm: 8, md: 12 }}
+        >
+          {brands && brands.map((brand) => (
```

Comment:
what if we had 20 brands, Will we render all of them?!

Reply:
done